tests/base/src/test/resources/filesystem-realm-exists/u/s/e/user-OVZWK4Q.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><identity xmlns="urn:elytron:identity:1.2">
+    <principal name="user"/>
+    <credentials>
+        <password algorithm="clear" format="base64">AXNlY3JldFBhc3N3b3Jk</password>
+    </credentials>
+    <Signature xmlns="http://www.w3.org/2000/09/xmldsig#">
```

Comment:
Just thinking about the formatting issue we just discussed. Is this formatting valid here?

Reply:
It isn't but this tests is checking an invalid signature. I thought it would be good for someone to be able to visualise the signature tag a bit better.
Should I set it to what the realm normally generates instead?

Reply:
Is the signature only invalid due to the formatting or due to the actual content too?

Reply:
The content is also invalid, seen here
https://github.com/wildfly-security/wildfly-elytron/blob/c4f7abc8a9db22eaad66bab555884bb60e3b5e17/tests/base/src/test/resources/filesystem-realm-exists/u/s/e/user-OVZWK4Q.xml#L18